style.css

```diff
@@ -531,3 +531,34 @@ body > * > * {
   margin-top: -7px;
   border-width: 7px 7px 7px 0;
 }
+
+details {
+  font: 16px;
+  width: 1000px;
+}
+
+details > summary {
```

Comment:
Rather than duplicate the button styles here we could apply the `.button-primary` class to that specific `summary` element.